### ISSUE TITLE
Rounds not starting due to the addon

### ISSUE THREAD

**Issue by dpaci** [lzinga/TTTWeightedTraitorSelection#6]
Hey, some days ago I had this addon and this ERROR appeared sometimes and it wouldn't let the rounds start unless an admin forced it to do so (through ULX and stuff). I had to remove the addon because I couldn't keep it like that but I'm really interested in having it since many people complain about base TTT's unfair way to select players' roles (one can get traitor 5 rounds in a row, another can get innocent 15 maps in a row if you get me).

Anyway I managed to get the error through my server's console (ignore the players' chat, it's the first three errors).
![ss 2016-11-21 at 08 16 26](https://cloud.githubusercontent.com/assets/23656588/20865243/284bdf70-ba0d-11e6-8195-f2f6346302e6.png)

**Comment by LaurenceKaye** [comment]
Does it do this randomly or is it always the first round on a map?

**Comment by dpaci** [comment]
I just installed the addon today on my server and it's working fine, maybe
I just touched some stuff back in the day but I'll tell you if the problem
persists. It used to do this on random maps in random rounds, while others
would work well other rounds would get stuck and all that jazz.

**Comment by dpaci** [comment]
It's alls good now, thank you! It was a mistake  on my side.

**Comment by LaurenceKaye** [comment]
It's odd that you would say that as this is a problem that I too have experienced.

How did you 'fix' it your end?

**Comment by dpaci** [comment]
Well when I firstly installed the addon I edited a bit some stuff to totally remove the !tc command but I guess I fucked it up since I'm not really a LUA professional; though I re-uploaded it and gave my server two days to see if the problems persisted, which did not.

All I did was simply using the convars to disable the !tc command and made sure I wouldn't touch even a character, just slide it in there flawlessly.

Though, it may have been just luck from my part, if it happens once again I won't hesitate typing it here. If it's of any help I'm not using the database stuff.
